.env_example

```diff
@@ -0,0 +1,8 @@
+DG_API_KEY=""
```

Comment:
These credentials are only used by the example scripts, right? If so, does this file need to be in the root directory, or could it live in `examples/`?